src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -276,15 +276,17 @@ export type ProjectKey = {
 export type Health = {
   totalUsers: number;
   totalUsers24h: number | null;
+  totalProjectUsers24h: number | null;
   totalSessions: number;
   totalSessions24h: number | null;
+  totalProjectSessions24h: number | null;
   crashFreeUsers: number | null;
   crashFreeSessions: number | null;
   stats: HealthGraphData;
   sessionsCrashed: number;
   sessionsErrored: number;
   adoption: number | null;
-  sessions_adoption: number | null;
+  sessionsAdoption: number | null;
```

Comment:
Is this field in use by existing client code? You might see some cross release errors as the old key will be undefined in the new API responses.

Reply:
This field is only a few days old and is not visible to GA (it's feature-flagged for ~5 sentry employees).